auslib/admin/views/forms.py

```diff
@@ -3,12 +3,16 @@
 from flask_wtf import Form
 from wtforms import StringField, IntegerField, SelectField
 from wtforms.widgets import TextInput, FileInput, HiddenInput
-from wtforms.validators import Required, Optional, NumberRange, Length
+from wtforms.validators import Required, Optional, NumberRange, Length, Regexp
 
 import logging
 log = logging.getLogger(__name__)
 
 
+# If present, rule alias' must be a string containing at least one non-numeric character.
+RULE_ALIAS_REGEXP = "(^.*[^0-9].*$|^$)"
```

Comment:
this is pretty complex. can we say an alias has to start with a letter, and contain only alphanumeric characters, plus maybe -? i.e. "[a-zA-Z][-a-zA-Z0-9]+" ?